### PR TITLE
GraphQL links added in sidebar

### PR DESCRIPTION
**Pull request by AnujSharma141**
Added GraphQL video to additional resources links in sidebar

### CODE REVIEW

**Comment by AnujSharma141** [comment]
Fix #2551 

Created a new PR to resolve branch conflicts of earlier one (#2594 )

File: src/pages/docs/sending-requests/supported-api-frameworks/graphql.md [hunk 1]
```diff
@@ -18,6 +18,11 @@ contextual_links:
   - type: link
     name: "Working with GraphQL template"
     url: "https://explore.postman.com/templates/1692/working-with-graphql"
+  - type: subtitle
+    name: "Video"
+  - type: link
+    name: "Working with GraphQL in Postman"
```
Comment (SueSmith) [inline src/pages/docs/sending-requests/supported-api-frameworks/graphql.md:24]: if we could use the same name as the title of the youtube video that would be perfect

Comment (AnujSharma141) [reply src/pages/docs/sending-requests/supported-api-frameworks/graphql.md]: @SueSmith Sure I will do that. Thanks a lot for suggesting.

**Comment by SueSmith** [comment]
Hey @AnujSharma141 thanks for all the great PRs 🙌 - plz fill out the [contributor form](https://docs.google.com/forms/d/e/1FAIpQLSfbLAcxl-IOiv3NmgEaWw7FleOaXnIyIoIrY_zn6U4JvjQBGA/viewform) so we can send you something cool 😎

**Comment by AnujSharma141** [comment]
Thank you @SueSmith !! That's Great 😃.

**Pull request merged**